docs/source/models/segment_anything.rst

```diff
@@ -9,7 +9,7 @@ can be used to generate masks for all objects in an image.
 
     **Segment Anything**
     ^^^
-    **Abstract:** We introduce the Segment Anything (SA) project: a new task, model, and dataset for image
```

Comment:
These changes, and some others are artifacts from the merge

Reply:
yes, i'll fix that soon